### PR TITLE
Update Nvidia metrics

### PR DESCRIPTION
**Pull request by jongiddy**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Update Nvidia Prometheus exporter and Grafana dashboard.

**Which issue(s) this PR fixes**:
https://jira.d2iq.com/browse/D2IQ-75525
https://jira.d2iq.com/browse/D2IQ-75529
https://jira.d2iq.com/browse/D2IQ-75528
https://jira.d2iq.com/browse/D2IQ-75526

**Special notes for your reviewer**:
This was tested using the procedure described in the `TESTING.md` file. To test, create a Konvoy cluster with GPU nodes according to https://docs.d2iq.com/dkp/konvoy/1.8/gpu/ and with the Konvoy `cluster.yaml` pointing to
```yaml
  - configRepository: https://github.com/mesosphere/kubernetes-base-addons
    configVersion: D2IQ-75525/bump-dcgm
```

See https://jira.d2iq.com/browse/D2IQ-75528 for some screenshots of the Grafana dashboard.

**Does this PR introduce a user-facing change?**:

```release-note
Updated Nvidua Prometheus outputs and Grafana dashboard.
```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 appVersion: "0.1.0"
 description: Nvidia GPU Metrics Exporter
 name: nvidia-dcgm-exporter
-version: 0.1.1
+version: 0.1.2
 maintainers:
-  - name: gilbert88
\ No newline at end of file
+  - name: gilbert88
```
Comment (makkes) [inline staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml:7]: As @Gilbert88 isn't with the company, anymore, you might want to think about adding yourself (and/or anyone you see appropriate) here.

**Review by makkes — APPROVED** [review]

File: staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml (hunk 1 above)
Comment (jongiddy) [reply staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml]: Done

File: staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 appVersion: "0.1.0"
 description: Nvidia GPU Metrics Exporter
 name: nvidia-dcgm-exporter
-version: 0.1.1
+version: 0.1.2
 maintainers:
-  - name: gilbert88
\ No newline at end of file
+  - name: jongiddy
+  - name: tillt
```
Comment (tillt) [inline staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml:8]: Thanks again :)

**Review by tillt — APPROVED** [review]
Thanks Jon - this adds great value already once we bump kba. The CVE reporter will be much happier as well :)

File: staging/nvidia/charts/nvidia-dcgm-exporter/templates/daemonset.yaml [hunk 3]
```diff
@@ -28,44 +28,47 @@ spec:
...
+          initialDelaySeconds: 5
       volumes:
       - name: pod-gpu-resources
         hostPath:
           path: /var/lib/kubelet/pod-resources
       - name: device-metrics
         emptyDir:
-          medium: Memory
\ No newline at end of file
+          medium: Memory
```
[the remark this answers is not in the document]
Comment (jongiddy) [reply staging/nvidia/charts/nvidia-dcgm-exporter/templates/daemonset.yaml]: Thanks. Good catch!

**Pull request merged**